content/en/docs/tasks/security/authorization/rbac-groups/index.md

```diff
@@ -168,6 +168,8 @@ wait until the newly defined authorization policy to take effect.
     EOF
     {{< /text >}}
 
+    Wait for the newly defined policy to take effect.
```

Comment:
Would it be a good idea to say slightly more about roughly how long they should expect to wait? (For example, is it more like seconds or more like weeks?) Fine to leave alone, of course.

Reply:
Fixed